package haptix.comm.msgs;

message hxQuaternion
{
  required float w = 1;
  required float x = 2;
  required float y = 3;
  required float z = 4;
}
